students/pedro-acuña/prueba1.html

```diff
@@ -0,0 +1,107 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <a name="Grogu" id="Grogu"></a>
+    <h1>Grogu siendo adorable</h1>
+    <img src="https://sm.ign.com/t/ign_latam/screenshot/default/babygrogu_tch1.2560.jpg" alt = "Grogu" width="600"/>
+    <h2>Por qué deberías ver The Mandalorian</h2>
+    <p><b>The Mandalorian</b> es una serie sobre el universo de <i>Star Wars</i> de estilo <u>Western.</u>
+    La serie trata de un cazarrecompensas Mandaloriano al cual seguimos por toda la galaxia sus aventuras por
+    diversos mundos donde la acción y la aventura no faltarán. También conoceremos a Grogu, un pequeño muy adorable
+    que nos enternecerá el corazón. </p>
+    <ul>
+        <li><a href = "https://www.google.es/">Google</a> </li>
+        <li><a href = "mailto: pedritociudad@hotmail.es">Email de contacto</a></li>
+        <li><a href= "tel:601244768">Teléfono de contacto</a></li>
```

Comment:
If this is your real phone. please remove it and use a fake one. There is spiders trying to collect your personal information